app/form_models/publishers/vacancies/job_applications/mark_reference_as_received_form.rb

```diff
@@ -1,18 +0,0 @@
-module Publishers
-  module Vacancies
-    module JobApplications
-      class MarkReferenceAsReceivedForm
-        include ActiveModel::Model
-        include ActiveModel::Validations
-        include ActiveModel::Attributes
-
-        ATTRIBUTES = [:reference_satisfactory].freeze
-
-        ATTRIBUTES.each do |field|
-          attribute field, :boolean
-          validates field, inclusion: { in: [true, false], allow_nil: false }
-        end
-      end
-    end
-  end
-end
```

Comment:
again I don't see the need to remove this class

Reply:
this step has been remove in new flow